manager/client.go

```diff
@@ -16,6 +16,10 @@ func (c *Client) ReceivePayload() <-chan event.Payload {
 	return c.payloadChan
 }
 
+func (c *Client) SetEvents(events []string) {
```

Comment:
__[golint]__
* [golint] exported method Client.SetEvents should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->